modules/aws/bastion/bastion-logs.tf

```diff
@@ -1,12 +1,14 @@
 resource "aws_cloudwatch_log_group" "bastion_log_group" {
-  name = "${var.cluster_name}_bastion"
+  count = "${ var.forward_logs_enabled ? 1 : 0 }"
```

Comment:
nitpick `${var.forward_logs_enabled ? 1 : 0 }`

because at some places you use with whitespace at some w/o